### ISSUE TITLE
关于训练过程中Target的请教

### ISSUE THREAD

**Issue by MstarLioning** [SpatialVLA/SpatialVLA#45]
您好！很抱歉再次打扰！我们当前想对SpatialVLA在["google_robot_pick_coke_can","google_robot_pick_horizontal_coke_can", "google_robot_pick_vertical_coke_can","google_robot_pick_standing_coke_can","google_robot_pick_object","google_robot_move_near_v0","google_robot_move_near_v1","google_robot_move_near"]这八项任务中进行梯度相关的解析，但是我不太确定您在SimplerEnv的模仿学习过程中每步使用的是什么来源的专家轨迹（之前查证了一下发现SImplerenv上不提供专家轨迹），所以想知道可以从哪里学习下您训练的过程或者可以从哪提取到每步的梯度/Target信息么？万分感激，期待您的回复！

**Comment by DelinQu** [comment]
您好！感谢关注本项目。SpatialVLA训练过程的loss曲线在https://arxiv.org/pdf/2501.15830 Fig.12。SimplerEnv的训练数据是https://huggingface.co/datasets/IPEC-COMMUNITY/bridge_orig_lerobot 和 https://huggingface.co/datasets/IPEC-COMMUNITY/fractal20220817_data_lerobot 对应的rlds格式，训练中间过程的梯度以及中间过程sft一般不向外提供的，只有最终训练好的权重。

**Comment by MstarLioning** [comment]
@DelinQu 收到，谢谢您的回复！当前还有一个问题不太明白：我们通过观察modeling_spatialvla.py后发现计算损失函数的方式是loss_fct = nn.CrossEntropyLoss()，而通过观察 loss = loss_fct(flat_logits, flat_labels) 后不知道在SpatialVLA中是如下这两种情况的哪种。
1、将专家轨迹的第n帧与实际得出的第n帧的7维Action序列做交叉熵
2、是将所有帧的的Action合并，再跟GroundingTruth做交叉熵

如果是第一种情况的话比如专家演示有13帧而模型实际跑了14帧感觉就有点对不齐了。但是如果是第二种的话似乎又没法逐帧矫正、计算损失和梯度。

希望能得到您的回复！万分感谢！
